assets/compat/btc-rpc-proxy_config_rules.yaml

```diff
@@ -119,13 +119,13 @@
     - PUSH:
         to: 'users.[first(item => ''item.name = "btcpayserver")].allowed-calls'
         value: "scantxoutset"
-- rule: '!((''[btcpayserver].bitcoin.bitcoind-rpc.type = "internal-proxy") AND (''[btcpayserver].bitcoin.bitcoind-rpc.pruned = "disabled")) OR users.[first(item => ''item.name = "btcpayserver")].fetch-blocks?'
+- rule: '!((''[btcpayserver].bitcoin.bitcoind-rpc.type = "internal-proxy") AND (''[bitcoind].advanced.pruning.mode = "disabled")) OR users.[first(item => ''item.name = "btcpayserver")].fetch-blocks?'
   description: RPC user "btcpayserver" must have "Fetch Blocks" disabled if bitcoind is a full archival node
```

Comment:
Does this work if bitcoind is not installed?

Reply:
not sure havent tested that yet - could you do so? my dev box is borked atm

i dont think it will. and i dont think it would have in the previous case either, since it depended on the pointer. 

we can't validate if an external node is pruned with this system.

Reply:
We should probably also recommend they whitelist their btcpay's IP address